storage/google/cloud/storage/blob.py

```diff
@@ -96,6 +97,12 @@
 _READ_LESS_THAN_SIZE = (
     "Size {:d} was specified but the file-like object only had " "{:d} bytes remaining."
 )
+_SIGNED_URL_V2_DEFAULT_MESSAGE = (
```

Comment:
You can remove this until we have a clear plan.

Reply:
ccccaed